boot/bootutil/src/image_rsa.c

```diff
@@ -88,20 +95,24 @@ bootutil_parse_rsakey(mbedtls_rsa_context *ctx, uint8_t **p, uint8_t *end)
         return -2;
     }
 
-    if ((rc = mbedtls_asn1_get_mpi(p, end, &ctx->N)) != 0 ||
-      (rc = mbedtls_asn1_get_mpi(p, end, &ctx->E)) != 0) {
+    if ((rc = mbedtls_asn1_get_mpi(p, end,
```

Comment:
In MCUboot there is no 80 char rule. So, might better to keep these are as a single line to improve readability.

Reply:
Done